tests/test_examples_app.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
```

Comment:
Note that the file name should be `test_example_app.py` (meaning we are testing example application...)
Can you please change? See https://github.com/inveniosoftware/cookiecutter-invenio-module/tree/master/%7B%7B%20cookiecutter.project_shortname%20%7D%7D/tests